content/docs/guides/(deployment)/deployment-options.mdx

```diff
@@ -13,8 +13,6 @@ UserClouds offers four deployment models to suit different needs and environment
 
 - **SaaS Deployment**: Use UserClouds in a pure SaaS environment with minimal infrastructure configuration. Perfect for companies that prefer a fully managed service with minimal overhead for configuration and maintenance.
 - **[VPC Peering Deployment](/docs/guides/set-up-guides/vpc-peering-set-up-guide)**: Set up UserClouds without exposing services to the internet. Best for organizations with high network security needs, who do not want to handle the complexity of running on-prem.
-- **[Docker Deployment](/docs/guides/set-up-guides/docker-set-up-guide)**: Deploy UserClouds using Docker for quick and easy setup. Note: because this configuration provides no monitoring, automatic restarts, failover, etc, this configuration is intended ONLY for development, test, and CI environments. Do not use this configuration for production - otherwise outages and data loss may result.
-- **[On-Premise K8s Deployment](/docs/guides/set-up-guides/eks-set-up-guide)**: Deploy UserClouds on your Kubernetes cluster for a fully managed, scalable solution. Suitable for large enterprises with existing Kubernetes infrastructure looking for a scalable and robust deployment option.
```

Comment:
we ship;d actually keeps this. since this is used by a customer.

Reply:
Just the write up on this page or these articles?

Reply:
we should keep the on-Premise K8s Deployment page/guide too.